### PR TITLE
Generate QR code for the deposit address

### PR DESCRIPTION
**Pull request by michalsmiarowski**
Closes: #287 

Generates proper QR Code for the deposit address.

For that purpose we will use [react-qr-code](https://www.npmjs.com/package/react-qr-code) lib.

### CODE REVIEW

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx [hunk 1]
```diff
@@ -22,10 +21,9 @@ import CopyToClipboard from "../../../../components/CopyToClipboard"
 import { useTbtcState } from "../../../../hooks/useTbtcState"
 import shortenAddress from "../../../../utils/shortenAddress"
 import { MintingStep } from "../../../../types/tbtc"
-
-import { Divider, useColorModeValue } from "@chakra-ui/react"
 import ViewInBlockExplorer from "../../../../components/ViewInBlockExplorer"
 import { ExplorerDataType } from "../../../../utils/createEtherscanLink"
+import QRCode from "react-qr-code"
```
Comment (r-czajkowski) [inline src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx:26]: Can we create our wrapper for this external component? If we changed the lib for generating QR code we would change only our wrapper component and keep the same interface.

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx [hunk 2]
```diff
@@ -77,23 +75,33 @@ export const MakeDeposit: FC = () => {
...
-        <Image
-          my={6}
-          mx="auto"
-          bg="white"
-          borderRadius="lg"
-          maxW="145px"
-          maxH="145px"
-          src={btcQrTmp}
-        />
+        <Flex direction={"column"} alignItems={"center"}>
```
Comment (r-czajkowski) [inline src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx:78]: Do we need flex wrapper?

**Review by r-czajkowski — CHANGES_REQUESTED** [review]
Left two minor comments.

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx (hunk 1 above)
Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx]: Fixed in 0bbe4fbc41e7953f1b58180c6e87c946bce03e9f

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx (hunk 2 above)
Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx]: Fixed in 5ae4f2febbc0c63b29f243f60d66a8a90da59b39

**Review by r-czajkowski — APPROVED** [review]

**Pull request merged**